pkgs/tools/package-management/nix/common.nix

```
@@ -246,6 +242,9 @@ self = stdenv.mkDerivation {
   # See https://github.com/NixOS/nix/issues/5687
   + lib.optionalString (atLeast25 && stdenv.isDarwin) ''
     echo "exit 99" > tests/gc-non-blocking.sh
+  '' + ''
+    # nixStatic otherwise does not find its man pages in tests otherwise
```

I know I'm nitpicking here, but 
```suggestion
    # nixStatic otherwise does not find its man pages in tests.
```